README.md

```diff
@@ -143,6 +143,10 @@ post_install do |installer|
 end
 ```
 
+#### Manually
+1. Download and drop ```/SwiftTweaks``` folder in your project.  
```

Comment:
These installation instructions aren't sufficient; there's a dependency on a xcconfigs submodule. See #61 for the submodule installation instructions that should be included as well.

Thanks!